src/ops.rs

```diff
@@ -62,6 +62,13 @@ pub fn jmp_abs(vm: &mut VM, pipeline: &Pipeline) -> Result<(), VMError>{
     Ok(())
 }
 
+pub fn jmp_conditional_ecx_is_zero(vm: &mut VM, pipeline: &Pipeline) -> Result<(), VMError> {
+    if vm.regs[1] == 0x0 {
```

Comment:
Change this to use the regs32.ECX constant for clarity. also you can just use `0` rather than 0x0